model/src/main/proto/analytics.proto

```diff
@@ -8,13 +8,13 @@ option java_multiple_files = true;
 // Supported event actions for analytics logging
 enum EventAction{
   // The undefined user action of an event
-  UNKNOWN_EVENT_ACTION = 0;
+  EVENT_ACTION_UNSPECIFIED = 0;
 }
 
 // Supported priority of events for analytics logging
 enum Priority{
   // The undefined priority of an event
-  UNKNOWN_PRIORITY = 0;
```

Comment:
@Sarthak2601 @vinitamurthi PTAL if this somehow effects your code or not.

Reply:
I think it should be fine, this is currently not used anywhere